### DISCUSSION TITLE
Leading zero is removed from template field

### DISCUSSION THREAD

**Discussion by calvinbui** [renovatebot/renovate#21153] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.21.0

### If you're self-hosting Renovate, select which platform you are using.

Gitea

### If you're self-hosting Renovate, tell us what version of the platform you run.

1.19.0

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am updating https://hub.docker.com/r/collabora/code/ to version `22.05.12.4.1`.

In the PR, I want to add a link to the release notes:

```json5
    {
      matchPackagePatterns: ["collabora\\/code"],
      versioning: "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)\\.(?<build>\\d+)\\.1$",
      prBodyNotes: "[Release Notes](https://www.collaboraoffice.com/code-{{newMajor}}-{{newMinor}}-release-notes/)",
    },
```

The release notes should go to: `https://www.collaboraoffice.com/code-22-05-release-notes/`

However, `{{newMinor}}` returns `5` instead of `05` so I get a link to `https://www.collaboraoffice.com/code-22-5-release-notes/`

Shouldn't `newMinor` return `05` if the version number is specifically `05` instead of `5`?

### What you tried so far.

N/A

### Relevant debug logs

N/A

**Comment by viceice** [comment, marked as answer]
expected because it's converted to number

**Reply by calvinbui** [reply]
could I use an `{{#if` or something to detect this and put it back?

**Reply by viceice** [reply]
probably yes

**Reply by calvinbui** [reply]
```
      prBodyNotes: "[Release Notes](https://www.collaboraoffice.com/code-{{newMajor}}-{{#if (containsString currentVersion '.0')}}0{{/if}}{{newMinor}}-release-notes/)",
```
